src/components/common.js

```diff
@@ -69,12 +69,12 @@ export const buttonPrimary = ({ disabled, ...props }, children) => {
     disabled,
     style: {
       ...styles.button,
-      border: `1px solid ${disabled ? colors.gray[4] : colors.green[0]}`,
+      border: `1px solid ${disabled ? colors.dark(0.4) : colors.primary()}`,
```

Comment:
The buttonPrimary coloring is much paler, and looks less like a CTA:
<img width="488" alt="Screen Shot 2019-05-17 at 8 02 33 AM" src="https://user-images.githubusercontent.com/2395211/57926768-2692aa00-787a-11e9-974a-33d9c119d972.png">

currently:
<img width="486" alt="Screen Shot 2019-05-17 at 8 03 13 AM" src="https://user-images.githubusercontent.com/2395211/57926796-3ca06a80-787a-11e9-9fad-799b7afdc1b1.png">